adapter_pipelines/submit.wdl

```diff
@@ -9,6 +9,7 @@ task get_metadata {
   Int? individual_request_timeout
   Boolean use_caas
   Boolean record_http
+  Int max_retries = 0
```

Comment:
So how do we tweak the numbers here? we change the static input json or we pass it in from Lira?

Reply:
It gets passed in from the Lira config